File: __tests__/__fixtures__/output/node/request/text-plain.js
const request = require('request');

const options = {
  method: 'POST',
  url: 'https://httpbin.org/anything',
  headers: {'content-type': 'text/plain'},
  body: 'Hello World'
};

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

File: __tests__/__fixtures__/output/csharp/httpclient/text-plain.cs
var client = new HttpClient();
var request = new HttpRequestMessage
{
    Method = HttpMethod.Post,
    RequestUri = new Uri("https://httpbin.org/anything"),
    Content = new StringContent("Hello World")
    {
        Headers =
        {
            ContentType = new MediaTypeHeaderValue("text/plain")
        }
    }
};
using (var response = await client.SendAsync(request))
{
    response.EnsureSuccessStatusCode();
    var body = await response.Content.ReadAsStringAsync();
    Console.WriteLine(body);
}